requirements-doc.txt

```diff
@@ -0,0 +1 @@
+pygit2<=1.10
```

Comment:
why are you creating a new file?

Reply:
My bad, I was too fast and edited (hence created) wrong path